src/components/cards/controltable/ControlRowDetails.vue

```diff
@@ -222,8 +242,12 @@ export default class ControlRowDetails extends ControlRowDetailsProps {
     ].filter(v => v.value); // Get rid of nulls
   }
 
+  //for zebra background
   zebra(ix: number): string {
-    return ix % 2 ? '' : 'zebra-table';
+    if (ix % 2 == 0) {
+      return 'zebra-table';
+    }
+    return 'non-zebra-table';
```

Comment:
Why was the ternary operator replaced here? `return ((ix % 2 == 0) ? 'non-zebra-table' : 'zebra-table');` should be equivalent.